src/stores/rootStore.ts

```diff
@@ -196,6 +196,11 @@ class RootStore {
   private lastOrDefaultInstrument = (defaultInstrument: any) => {
     try {
       const lastUsedInstrument = JSON.parse(instrumentStorage.get()!);
+
+      if (!lastUsedInstrument) {
```

Comment:
not really the scope of this request, but a just a small optimization I need to apply before I forget